eve/methods/common.py

```diff
@@ -1047,3 +1047,23 @@ def oplog_push(resource, document, op, id=None):
 
     if entries:
         app.data.insert(config.OPLOG_NAME, entries)
+
+    def _client_projection(self, req):
```

Comment:
what I mean is that your PR has two `_client_projection` methods. One is in `eve.io.base` and another in `eve.methods.common`, which appears to be not only superfluous but also orphaned. I think that the latter should be removed (probably a leftover from experimenting with the code move)